WalletWasabi.Fluent/Styles/TextBox.axaml

```diff
@@ -162,8 +162,18 @@
     </Setter>
   </Style>
 
-  <Style Selector="TextBlock.monoSpaced">
-    <Setter Property="FontFamily" Value="{StaticResource MonospacedFont}" />
-    <Setter Property="FontSize" Value="15" />
+  <Style Selector="TextBox.readOnly">
```

Comment:
We already have a `selectableTextBlock` class, which seemingly does the same except for the background. Could you take a look at it? Perhaps they could be merged into one.